core/src/i18n/en.yaml

```diff
@@ -916,7 +916,7 @@ destination-rules:
 discovery-and-network:
   title: Discovery and Network
 dnsentries:
-  description: <0>DNS Entry</0> is an object that represents the request to create an external DNS record.
+  description: <0>DNSEntry</0> is an object that represents the request to create an external DNS record.
   headers:
     dns-name: DNS Name
```

Comment:
Additionally - shouldn't it be `DNS name` in lines 921, 929? Similar with line 923